pkg/collector/systemperf_collector.go

```diff
@@ -22,23 +42,73 @@ func (collector *SystemPerfCollector) GetName() string {
 
 // Collect implements the interface method
 func (collector *SystemPerfCollector) Collect() error {
-	if err := utils.CreateKubeConfigFromServiceAccount(); err != nil {
```

Comment:
💡 I am wondering what was this function for and the new `client-go` takes care of it?

Reply:
Needed to set the config used by kubectl in prior implementation. The metrics.newForConfig below would be equivalent in client-go